static/app/views/organizationStats/teamInsights/overview.tsx

```diff
@@ -352,7 +352,7 @@ const Body = styled(Layout.Body)`
 const ControlsWrapper = styled('div')`
   display: grid;
   align-items: center;
-  gap: ${space(1)};
+  gap: ${space(2)};
```

Comment:
I just changedhttps://github.com/getsentry/sentry/pull/30490 wait I just changed releases spacing here.

`space(1)` seems more consistent with what the rest of the app uses no?

Reply:
Nah, it should be `space(2)`— Team Stats, Issues, and Alerts are the outliers. We’ll get around to fixing it there, too. Releases should stick to `space(2)` horizontal/vertical gap as well.